src/Toml/BiMap.hs

```diff
@@ -157,31 +161,42 @@ _ShowString = BiMap (Just . show) readMaybe
 _Show :: (Show a, Read a) => BiMap a AnyValue
 _Show = _ShowString >>> _String
 
-_UnsignedInteger :: (Integral n) => BiMap n Integer
-_UnsignedInteger = prism fromIntegral maybeInteger
+_NaturalInteger :: BiMap Natural Integer
+_NaturalInteger = BiMap (Just . toInteger) maybeInteger
   where maybeInteger n
```

Comment:
I would appreciate having explicit type signature for `maybeInteger` function to make it clear in what direction numbers are moving. It's quite easy to swap arguments of `_NaturalInteger` bimap and not notice it... So something like this really improves code readability:

```haskell
  where
    maybeInteger :: Integer -> Maybe Natural
    maybeInteger n
         | n < 0 ....
```